db/schema.rb

```diff
@@ -1499,6 +1500,16 @@
 
   add_index "videos", ["episode_id"], name: "index_videos_on_episode_id", using: :btree
 
+  create_table "volumes", force: :cascade do |t|
+    t.string   "title",      null: false
+    t.string   "isbn",       null: false
```

Comment:
Teeeeeeechnically, Postgres has a module with an ISBN type, but reading over the docs I guess barcodes are a mess and we'll probably need a library for that when we wanna do more with it

So string is probably a good choice until we actually know more about what we're doing :+1: